### DISCUSSION TITLE
Update 8 is now out on Early Access!

### DISCUSSION THREAD

**Discussion by konstander** [wolveix/satisfactory-server#214] [General]
Greetings! Is the current build of the dedicated server compatible with the new Update 8, which was released on November 14 in early access?
Have you heard anything from the developers about their resumption of work on dedicated servers? In the summer, I heard that all forces were directed to the release of update 8. And here we are at the line...

**Comment by konstander** [comment]
Have you noticed any unpleasant bugs that it is better not to play with yet and wait for updates in the server part?

**Reply by wolveix** [reply]
@konstander this repo has worked with update 8 since it's released on experimental, the release for early access makes no difference :)

**Reply by konstander** [reply]
<html>
<body>

STEAMBETA | false or true?



</body>
</html>

**Reply by wolveix** [reply]
`STEAMBETA` is only for experimental. If you're playing early access, then it should be set to `false`

**Comment by konstander** [comment]
> `STEAMBETA` is only for experimental. If you're playing early access, then it should be set to `false`
The issue is resolved.
